core/src/main/scala/cats/data/XorT.scala

```diff
@@ -219,6 +219,11 @@ trait XorTFunctions {
 
   final def pure[F[_], A, B](b: B)(implicit F: Applicative[F]): XorT[F, A, B] = right(F.pure(b))
 
+  /**
+   * Alias for [[XorT.right]]
+   */
+  final def liftT[F[_], A, B](fb: F[B]): XorT[F, A, B] = right(fb)
```

Comment:
That was my bad, I pushed up a bad commit with the scaladoc link in it.  I ended up in a hurry and my laptop was dieing.  I'll get that all fixed along with the sbt-doctest later this evening / tomorrow